packages/gafl-webapp-service/src/processors/pricing-summary.js

```diff
@@ -114,7 +131,13 @@ export const pricingDetail = async (page, permission) => {
               }),
               {}
             )
-          return { [licenceType]: Object.assign(filtered, Object.keys(filtered).length < 3 ? { msg: NO_SHORT } : {}) }
+          return {
+            [licenceType]: Object.assign(
+              filtered,
+              Object.keys(filtered).length < length ? { msg: NO_SHORT } : {},
```

Comment:
I appreciate that 3 is a magic number in the original code, but I don't think that extracting it to a variable named 'length' really adds any clarity. Leave it as it was, unless you have a clear understanding of why it's 3 in which case give the variable a better name :)